internal/pkg/cli/project_show.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// ShowProjectOpts contains the fields to collect for showing a project.
+type ShowProjectOpts struct {
```

Comment:
Does this struct need to be exported?

Reply:
That's a good question. I don't see any reason why it is exported but it seems like all the other structs in CLI package are exported. Maybe we should fix them.

Reply:
I'll change their names in a separate PR.